### PR TITLE
Myia CNN example and Policy Gradient Example

### PR DESCRIPTION
**Pull request by ethancaballero**
- also adds part of public_api (myia library)

### CODE REVIEW

File: myia/public_api.py [hunk 1]
```diff
@@ -388,21 +388,16 @@ def item(x):
...
-
     if dim is None:
         dim = _build_fwd_tuple(x.shape)
     elif isinstance(dim, int):
         dim = (dim,)
-    elif isinstance(dim, tuple):
-        pass
-    else:
-        assert keepdim is False
-        return P.array_map(P.scalar_max, x, dim)
```
Comment (abergeron) [inline myia/public_api.py]: This adds the 2-array compare case, why do you remove it?

File: myia/public_api.py [hunk 2]
```diff
@@ -21,13 +21,13 @@
 import numpy as np
 import torch
```
Comment (abergeron) [inline myia/public_api.py]: If this is to become the public myia API it can't have a dependency on PyTorch.

As it clearly still does, I don't approve of the rename.

**Comment by ethancaballero** [comment]
this is now subsumed by https://github.com/mila-iqia/myia/pull/340

**Comment by breuleux** [comment]
As Ethan said: subsumed by #340
